### DISCUSSION TITLE
A filter for Community Verified repos?

### DISCUSSION THREAD

**Discussion by adityaraute** [EddieHubCommunity/good-first-issue-finder#399] [Ideas]
Looking at the website I noticed we have two scopes of finding issues:
1) EddieHub
2) Everything else

![image](https://github.com/EddieHubCommunity/good-first-issue-finder/assets/43912470/c6d77033-1bac-47c8-a2df-dfef895bbf61)

I was wondering if adding a more moderate options, one where members of community can promote good-first-issues of their own repos, would be a good addition.
P.S. This idea came to me because I had a good-first-issue on one of the repos I'm maintaining.

**Comment by Cahllagerfeld** [comment]
You can already add your Good-First-issues. You need to assign them with a special tag see [the docs](https://finder.eddiehub.io/docs/your-issues). By assigning this label, they will be shown in the Github Section

**Comment by adityaraute** [comment]
Oh okay. In that case my suggestion is already present haha.

**Reply by Cahllagerfeld** [reply]
Yes, I'm closing the discussion for now 👍 Can you mark my answer als the solution please, so others can see it too 🙏

**Reply by adityaraute** [reply]
I can only upvote it. Don't see any option to mark it as a solution
